src/components/GraphFilter/LayoutDropdown.tsx

```diff
@@ -20,33 +21,49 @@ export class LayoutDropdown extends React.Component<LayoutDropdownProps, LayoutD
     ['dagre', 'Dagre'],
     ['klay', 'Klay']
   ];
+  static InitialLabel = 'Select Graph Type';
+
   constructor(props: LayoutDropdownProps) {
     super(props);
     this.state = {
-      currentGraphType: props.initialLayout
+      currentGraphType: props.initialLayout,
+      currentGraphTypeLabel: this.getGraphTypeName(props.initialLayout)
     };
   }
 
+  getGraphTypeName = (graphType: string) => {
+    let label = LayoutDropdown.InitialLabel;
+    LayoutDropdown.GraphTypes.map(subarray => {
+      if (subarray[0] === graphType) {
+        label = subarray[1];
+      }
+    });
+    return label;
+  };
+
   layout = (graphType: string) => {
-    this.setState({ currentGraphType: graphType });
+    this.setState({ currentGraphType: graphType, currentGraphTypeLabel: this.getGraphTypeName(graphType) });
     this.props.onClick(graphType);
   };
 
   render() {
     return (
-      <DropdownButton id="graphType" title="Type of Graph" onSelect={this.layout}>
-        {LayoutDropdown.GraphTypes.map(r => (
-          <MenuItem
-            key={r[0]}
-            active={r[0] === this.state.currentGraphType}
-            id={r[0]}
-            eventKey={r[0]}
-            disabled={this.props.disabled}
-          >
-            {r[1]}
-          </MenuItem>
-        ))}
-      </DropdownButton>
+      <div className="form-group">
+        <label style={{ paddingRight: '0.5em' }}>Graph Type:</label>
```

Comment:
Minor, but I think 'Graph Layout' or just 'Layout' may a better label.  We're not really changing the type of information presented in the graph, only the layout.  @mtho11 What do you think?

Reply:
@jshaughn yes, I agree `Layout` is more precise. And we are already on the Service Graph page so `Graph` is probably redundant.